utils/smstools3/files/smstools3.init

```diff
@@ -1,43 +1,22 @@
 #!/bin/sh /etc/rc.common
-# Copyright (C) 2014 OpenWrt.org
-# smsd initscript openwrt mod
+# Copyright (C) 2014-2018 OpenWrt.org
 
 START=99
 STOP=99
```

Comment:
Really? I think this can be removed.

Reply:
Indeed, this value doesn't make much sense. However what is the behaviour when STOP= is removed completly?

Reply:
STOP is optional. Set it to 10, 15 or remove it ... you haven't defined any special stop service procedure and your working directories are on tmpfs (reset after reboot anyway). Please test your package again after these changes - thanks.